### ISSUE TITLE
Large gap between released mAP and my exp's.

### ISSUE THREAD

**Issue by dreamPoet** [YueLiao/PPDM#39]
Thanks for your excellent work. I am trying to achieve the released mAP for DLA34 with hoidet_hico_dla.sh, however, after 140 epoch training I only get the result as below:

I run in python 3.6.12 and PyTorch 0.4.1. Would you mind giving me some tips about the reasons for the low performance?

Fix size testing.
training chunk_sizes: [32]
The output will be saved to  ../../exp/hoidet/hoidet_hico_dla
heads {'hm': 80, 'wh': 2, 'hm_rel': 117, 'sub_offset': 2, 'obj_offset': 2, 'reg': 2}
Namespace(K=100, arch='dla_34', batch_size=32, cat_spec_wh=False, chunk_sizes=[32], data_dir='../../data', dataset='hico', debug=0, debug_dir='/exp/hoidet/hoidet_hico_dla/debug', debugger_theme='white', demo='', dense_wh=False, down_ratio=4, exp_dir='../../exp/hoid
et', exp_id='hoidet_hico_dla', fix_res=True, flip=0.5, flip_test=False, gpus=[0], gpus_str='0', head_conv=256, heads={'hm': 80, 'wh': 2, 'hm_rel': 117, 'sub_offset': 2, 'obj_offset': 2, 're
g': 2}, hide_data_time=False, hm_weight=1, image_dir='hico_20160224_det/images/test2015', input_h=512, input_res=512, input_w=512, keep_res=False, load_model=''
, lr=0.000125, lr_step=[90, 120], master_batch_size=32, mean=array([[[0.40789655, 0.44719303, 0.47026116]]], dtype=float32), metric='loss', mse_loss=False, nms=False, no_color_aug=False, no
rm_wh=False, not_cuda_benchmark=False, not_prefetch_test=False, not_rand_crop=False, not_reg_offset=False, num_classes=80, num_classes_verb=117, num_epochs=140, num_iters=-1, num_stacks=1,
num_workers=4, off_weight=1, output_h=128, output_res=128, output_w=128, pad=31, print_iter=0, reg_loss='l1', reg_offset=True, resume=False, root_dir='lib/..
/..', root_path='../Dataset', rotate=0, save_all=False, save_dir='../../exp/hoidet/hoidet_hico_dla', save_predictions=False, save_video='', scale=0.4, se
ed=317, shift=0.1, std=array([[[0.2886383 , 0.27408165, 0.27809834]]], dtype=float32), task='hoidet', test=False, test_dir='', test_scales=[1.0], test_video=False, test_with_eval=True, trai
nval=False, use_cos=0, use_verb_sub=0, val_intervals=100000, vis_thresh=0.3, wh_weight=0.1)
Creating model...
loaded ../../exp/hoidet/hoidet_hico_dla/model_140.pth, epoch 140
----epoch :140 -----
--------------------
mAP: 0.19771157974547146 mAP rare: 0.11901581021109064  mAP nonrare: 0.2212181083076891  max recall: 0.49568942049889964
--------------------

**Comment by YueLiao** [comment]
The mAP looks only a 0.2% gap with our released results (19.77%->19.93%). It may be caused by the random seed or you can evaluate it by the official evaluation scripts.
![image](https://user-images.githubusercontent.com/30544357/103391191-0a2ca980-4b53-11eb-8e89-6025f2cdd20f.png)

**Comment by dreamPoet** [comment]
But I found there is a 60+% in the paper for mAP, what is that?

**Comment by YueLiao** [comment]
> But I found there is a 60+% in the paper for mAP, what is that?

I think you should read the paper carefully. 60+% is for the HOI-A dataset, and 20% is for HICO-Det.

**Comment by dreamPoet** [comment]
> > But I found there is a 60+% in the paper for mAP, what is that?
> 
> I think you should read the paper carefully. 60+% is for the HOI-A dataset, and 20% is for HICO-Det.

Sorry I used anno jsons you provided and just found they are "HOI-A format annotations " instead of "HOI-A annotations ".
Thank you so much.
